nntrainer/layers/acti_func.cpp

```diff
@@ -261,4 +269,21 @@ float ActiFunc::reluPrime(float x) {
 float ActiFunc::no_op(float x) { return x; }
 
 float ActiFunc::no_op_prime(float x) { return 1.0f; }
+
+constexpr static inline float NEGATIVE_SLOPE = 0.01f;
```

Comment:
i recommend making this a property as this is usually configurable.

Reply:
Yes, this need be. For now this is an improvised version to support current needs. (This has to be supported by monday).

Please refer to https://github.com/nnstreamer/nntrainer/pull/1742/files#r756727752

Reply:
cool.